src/lib/Gen/xml/XMLWriter.cpp

```diff
@@ -381,6 +385,81 @@ writeGuide(
     closeTemplate(I.Template);
 }
 
+void
+XMLWriter::
+writeAlias(
+    AliasInfo const& I)
+{
+    tags_.open(aliasTagName, {
+        { "name", I.Name },
+        { I.Access },
+        { I.id }
+        });
+
+    writeSourceInfo(I);
+
+    writeJavadoc(I.javadoc);
+
+    if (I.AliasedSymbol)
+    {
+        Attributes nameAttrs = {};
+        nameAttrs.push({"name", toString(*I.AliasedSymbol)});
+        tags_.write("name", {}, nameAttrs);
+    }
```

Comment:
```suggestion
    if (I.AliasedSymbol)
    {
        tags_.write("aliased", {}, {
            {"name", toString(*I.AliasedSymbol)},
            { I.AliasedSymbol->id }
        });
    }
```

I don't think `I.AliasedSymbol` will ever be null, so you can probably remove the branch too

Reply:
@sdkrystian I am having cases where `I.AliasedSymbol` is null